lightning_examples/warp-drive/multi_agent_rl.py

```diff
@@ -0,0 +1,343 @@
+#
+# We invite everyone to **contribute to WarpDrive**, including adding new multi-agent environments, proposing new features and reporting issues on our open source [repository](https://github.com/salesforce/warp-drive).
+
+
+# %%
+import logging
+
+import matplotlib.pyplot as plt
+import mpl_toolkits.mplot3d.art3d as art3d
+import numpy as np
+import torch
+from example_envs.tag_continuous.tag_continuous import TagContinuous
+
+# from IPython.display import HTML
+from matplotlib import animation
+from matplotlib.patches import Polygon
+from pytorch_lightning import Trainer
+from warp_drive.env_wrapper import EnvWrapper
+from warp_drive.training.pytorch_lightning_trainer import CudaCallback, PerfStatsCallback, WarpDriveModule
+
+# %%
+_NUM_AVAILABLE_GPUS = torch.cuda.device_count()
+assert _NUM_AVAILABLE_GPUS > 0, "This notebook needs a GPU to run!"
+
+# %%
+# Set logger level e.g., DEBUG, INFO, WARNING, ERROR.
+logging.getLogger().setLevel(logging.ERROR)
+
+# %% [markdown]
+# ## Specify a set of run configurations for your experiments
+#
+# The run configuration is a dictionary comprising the environment parameters, the trainer and the policy network settings, as well as configurations for saving.
+#
+# For our experiment, we consider an environment wherein $5$ taggers and $100$ runners play the game of [Tag](https://github.com/salesforce/warp-drive/blob/master/example_envs/tag_continuous/tag_continuous.py) on a $20 \times 20$ plane. The game lasts $200$ timesteps. Each agent chooses it's own acceleration and turn actions at every timestep, and we use mechanics to determine how the agents move over the grid. When a tagger gets close to a runner, the runner is tagged, and is eliminated from the game. For the configuration below, the runners and taggers have the same unit skill levels, or top speeds.
+#
+# We train the agents using $50$ environments or simulations running in parallel. With WarpDrive, each simulation runs on sepate GPU blocks.
+#
+# There are two separate policy networks used for the tagger and runner agents. Each network is a fully-connected model with two layers each of $256$ dimensions. We use the Advantage Actor Critic (A2C) algorithm for training. WarpDrive also currently provides the option to use the Proximal Policy Optimization (PPO) algorithm instead.
+
+# %%
+run_config = dict(
+    name="tag_continuous",
+    # Environment settings.
+    env=dict(
+        num_taggers=5,  # number of taggers in the environment
```

Comment:
```suggestion
        # number of taggers in the environment
        num_taggers=5,
```
I would do for all bellow so the notebook does ot have too long lines

Reply:
Done.